lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -968,6 +968,13 @@ export interface INavigationClient {
 
 export { INetworkModule }
 
+// Warning: (ae-missing-release-tag) "InitializeApplicationRequest" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Am I missing something here?  We're declaring this as @public on 973?

Reply:
We are declaring it as public in order to use it in 1p